src/routes/businessClass.js

```diff
@@ -49,7 +48,13 @@ router.post(
 );
 
 // read routes
-router.get("/:businessClassId", getBusinessClass);
+router.get(
+  "/:businessClassId",
+  isAuthorized(CLASS_DEFINITION, READ, {
```

Comment:
allow all authenticate user to read the class records, in other words set isAuthorized(null, null). Parent won't be able to find classes if we restrict them.